test/videojs-hls_test.js

```diff
@@ -1296,6 +1296,28 @@ test('blacklists switching from video-only playlists to video+audio', function()
   equal(videoAudioPlaylist.excludeUntil, Infinity, 'excluded incompatible playlist');
 });
 
+test('loadedMedia event only fire once we have successfully loaded a playlist', function() {
```

Comment:
Name should be `loadedmetadata`. How about "After an initial media playlist 404s, we fire loadedmetadata once we successfully load a playlist"? :)